src/main/java/frc/robot/subsystems/UnitModel.java

```diff
@@ -1,7 +1,7 @@
 package frc.robot.subsystems;
 
 public class UnitModel {
-    double ticksPerUnit;
+    private final double ticksPerUnit;
```

Comment:
Don't make changes like this in this PR.

Reply:
Why not?

Reply:
This PR cleans the code (at least that's what the title says). I don't think that a cleaning change.

Reply:
Not anymore